analysis/analysis.py

```diff
@@ -92,6 +93,38 @@ def run_analysis():
     show_response_groups(response_groups_freq_dicts)
     total_view_time = compute_total_view_time(student_data)
     print(f'The total view time of all students was {total_view_time}.')
+    print(count_revists(student_data))
+
+
+def count_revists(data):
+    """
+    Returns the average number of revisits per question
+
+    :param data: list, student response dict
+    :return: dict, Key = question, string. Value = average number of revisits, float.
+    """
+    results = {}
+
+    # Accumulates the total views and number of responses per unique question
+    for entry in data:
+        question = entry['question']
+        num_views = len(entry['views'])
+        result = results.get(question)
+        if result:
+            view_count, view_sum = result
+            view_count += 1
+            view_sum += num_views
+            results[question] = [view_count, view_sum]
+        else:  # Create a key with starting values
+            results[question] = [1, num_views]
+
+    # Averages the number of revisits per unique question
+    for question in results:
+        total_count, total_views = results[question]
+        views_per_count = total_views/total_count
```

Comment:
spaces around ```/``` -- I'm surprised pylint let this through! will check what's going on there.

Reply:
Fixed!